lib/Conversion/TorchToStd/TorchToStd.cpp

```diff
@@ -50,6 +50,20 @@ class ConvertAtenDimOp : public OpConversionPattern<AtenDimOp> {
 };
 } // namespace
 
+namespace {
+class Convert__Contains__IntOp : public OpConversionPattern<RuntimeAssertOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(RuntimeAssertOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    rewriter.replaceOpWithNewOp<cf::AssertOp>(op, adaptor.condition(),
+                                              adaptor.message());
+    return success();
+  }
+};
+} // namespace
+
```

Comment:
This code is completely wrong. You have just copy-pasted it from the `RuntimeAssertOp`. You need to modify this to work for the op you are working on. Also add a conversion pattern like this https://github.com/llvm/torch-mlir/blob/8ff03257df6c3a5841034b98bd2feb9e928df71e/lib/Conversion/TorchToStd/TorchToStd.cpp#L318-L319 for the `__contains__.int_list` op.

Reply:
We shouldn't need TorchTo* lowering for this op -- we don't support lists at runtime currently.

Reply:
Then adding the canonicalization/folding pattern in `TorchOps.cpp` would suffice?

Reply:
Yes.